components/form-builder/app/shared/Dialog.tsx

```diff
@@ -56,16 +58,14 @@ export const Dialog = ({
   }, [close]);
 
   return (
-    <dialog
-      className="p-0 m-0 bg-clip-padding w-full h-full bg-transparent"
-      aria-labelledby="modal-title"
```

Comment:
Removed this intentionally. I don't think this ID exists. Below an H2 is used if a title is present